mobile/library/cc/engine_builder.cc

```diff
@@ -303,6 +313,13 @@ std::string EngineBuilder::generateConfigStr() const {
     insertCustomFilter(filter_config, config_template);
   }
 
+  if (!rtds_layer_name_.empty()) {
+    std::string custom_layers_ = fmt::format(rtds_layer_insert, rtds_layer_name_, rtds_layer_name_,
```

Comment:
trailing underscores on variable names are only meant for class members, but see note above about making custom_layers_ the class variable instead of a local one